tools/ci_build/build.py

```diff
@@ -803,16 +805,16 @@ def main():
         cmake_extra_args = []
         if(is_windows()):
```

Comment:
Please add a TODO: need to check if cuda_version >=10.1, if cuda is enabled and VS version >=2019

Reply:
Added on line 494